docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -241,16 +253,98 @@ To update an existing network with a new `blockperiodseconds`:
             }
             ```
 
-1. Restart all nodes in the network using the updated genesis file.
-1. To verify the changes after the transition block, call
+3. Restart all nodes in the network using the updated genesis file.
+4. To verify the changes after the transition block, call
    [`ibft_getValidatorsByBlockNumber`](../../../Reference/API-Methods.md#ibft_getvalidatorsbyblocknumber), specifying `latest`.
 
-!!! caution
+### Update block rewards on an existing network deployment
```

Comment:
The block period uses the title "Configure block time...". Should these be consisent, i.e. either both use "Configure..." or both use "Update..."?